Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -42,19 +42,40 @@ function getCommandOutputFormat(kubecommand: string) : string[] {
     return args;
 }
 
-function getCommandConfigurationFile() : string[] {
-    var args: string[] =[];
-    var useConfigurationFile : boolean  =  tl.getBoolInput("useConfigurationFile", false);
-    if(useConfigurationFile) {
-        var configurationPath = tl.getInput("configuration", true);
-        if(configurationPath && tl.exist(configurationPath))
-        {
+function getCommandConfigurationFile(): string[] {
+    var args: string[] = [];
+    var useConfigurationFile: boolean = tl.getBoolInput("useConfigurationFile", false);
+    let configurationPath = tl.getPathInput("configuration", false);
+    var inlineConfiguration = tl.getInput("inline", false);
+
+    if (!tl.filePathSupplied("configurationPath")) {
+        configurationPath = null;
+    }
+
+    if (useConfigurationFile && configurationPath == null && inlineConfiguration == null) {
+        throw new Error(tl.loc('InvalidConfiguration', configurationPath, inlineConfiguration));
+    }
+    else if (configurationPath != null && inlineConfiguration != null) {
```

Comment:
Missing `useConfigurationFile` check here

Reply:
These changes are made to ignore useConfigurationFile flag if either configurationPath or inlineConfiguration is defined in YAML definition. So I assumed that it is an invalid state to have both configurations defined irrespective of useConfigurationFile

Reply:
Then you shouldn't be checking this in the if statement either, right?